### ISSUE TITLE
Random event question with wrong answer

### ISSUE THREAD

**Issue by 777btw** [oldschoolgg/oldschoolbot#2728]
## 🐛 Bug Report

Drunken Dwarf asks "What's the only monster that always drops an Ancient shard?"

There are none. 

Source:
https://oldschool.runescape.wiki/w/Skotizo#100%

**Comment by mylife212** [comment]
Skotizo is guaranteed to drop an ancient shard, as seen in the Wiki link you provided. Could reword it to make it clear it means any amount of an ancient shard if needed though.
![image](https://user-images.githubusercontent.com/63373565/131236023-b7c05836-59f0-4d11-bb83-2f1d6a3fe1d7.png)

**Comment by 777btw** [comment]
> Skotizo is guaranteed to drop an ancient shard, as seen in the Wiki link you provided. Could reword it to make it clear it means any amount of an ancient shard if needed though.
> ![image](https://user-images.githubusercontent.com/63373565/131236023-b7c05836-59f0-4d11-bb83-2f1d6a3fe1d7.png)

![image](https://user-images.githubusercontent.com/62408164/131236093-03c7e726-a61e-4852-8f0a-df0e576fad35.png)

You're right. My bad. Then wiki should also make it clear that it's in the 100% drop table
